### PR TITLE
feat: add gpts chat api

### PR DESCRIPTION
**Pull request by Abirdcfly**
#### What type of PR is this?
/kind bug
/kind feature

#### What this PR does / why we need it
1. add showRespInfo / showRetrievalInfo / showNextGuide /  enableUploadFile / notReadyReasonCode to gpts
2. add /gpts/chat api for gpts, which only check user token is valid
#### Which issue(s) this PR fixes

For #933 

@zggmd front web ui also need update.

### CODE REVIEW

File: apiserver/pkg/auth/auth.go [hunk 1]
```diff
@@ -178,6 +178,38 @@ func AuthInterceptorInGraphql(needAuth bool, oidcVerifier *oidc.IDTokenVerifier)
...
+		bearerToken := ctx.GetHeader("Authorization")
+		rawToken, ok := isBearerToken(bearerToken)
+		if !ok {
+			ctx.AbortWithStatusJSON(http.StatusUnauthorized, gin.H{
+				"message": "unauthorized, not bearer token",
+			})
+			return
+		}
+
+		oidcIDtoken, err := oidcVerifier.Verify(context.TODO(), rawToken)
```
Comment (bjwswang) [inline apiserver/pkg/auth/auth.go:196]: Use context from *gin.Context

File: apiserver/pkg/gpt/gpt.go [hunk 2]
```diff
@@ -47,18 +47,72 @@ func app2gpt(app *v1alpha1.Application, c client.Client) (*generated.Gpt, error)
...
+	groupKind, detail, find := strings.Cut(conditionMessage, " || ")
+	if !find {
+		return ConfigError
+	}
+	group, kind, find := strings.Cut(groupKind, ":")
+	if !find {
+		return ConfigError
+	}
+	switch group {
+	case "":
```
Comment (bjwswang) [inline apiserver/pkg/gpt/gpt.go:96]: why not "arcadia.kubeagi.k8s.com.cn" ?

Comment (Abirdcfly) [reply apiserver/pkg/gpt/gpt.go:96]: pkg/appruntime/base/node.go:71 has cut "arcadia.kubeagi.k8s.com.cn"
IMO, the following structure is clearer:

```go
	switch baseNode.Group() {
	case "chain":
		switch baseNode.Kind() {
		case "llmchain":
```

If we want to change it, we do it separately in a subsequent pr

Comment (bjwswang) [reply apiserver/pkg/gpt/gpt.go:96]: That make sense to me .

**Review by bjwswang — APPROVED** [review]

**Pull request merged**